docs/walkthrough.md

```diff
@@ -0,0 +1,553 @@
+
+Where did `pages` come from? That would be the template context processor you added earlier.
+`pages.current` refers to the currently active Page.
+`pages.current.content` refers to the instance of your content model that is attached to the page.
+
+This is the very simplest example of rendering a page's content model on the front end.
+There's all sorts of other things we can do with content models, but if your model has some fields, and maybe some inlines, you don't _need_ to write any views, just a template.
+
+Did we (royal we) say inlines? We definitely did.
+
+## Lets add some admin inlines
+
+OK, so let's say you want to build your new content page out of an entirely arbitrary number of sections.
+That's fine too!
+We can add inlines to our change-page view in the admin.
+Here is what your model looks like.
+Pay special attention to the ForeignKey if nothing else - this is essential, and note that it is to `pages.Page` and _not_ your content model:
+
+```
+class ContentSection(models.Model):
+    page = models.ForeignKey(
+        'pages.Page',
+        on_delete=models.CASCADE,
+    )
+
+    title = models.CharField(
+        max_length=100,
+    )
+
+    text = TextField(
+        null=True,
+        blank=True,
+    )
+
+    order = models.PositiveIntegerField(
+        default=0,
+    )
+
+    class Meta:
+        ordering = ['order']
+```
+
+We've defined a section model with a title, text, and an ordering field.
+Now let's register it as an inline for MyContent:
+
+```
+from cms.apps.pages.admin import page_admin
+from django.contrib.admin import StackedInline
+
+from .models import MyContent, ContentSection
+
+
+class ContentSectionInline(StackedInline):
+    model = ContentSection
+
+page_admin.register_content_inline(MyContent, ContentSectionInline)
+```
+
+That's it! It's just as easy as adding inlines to any other model.
+We've told the CMS "be prepared to display these inlines only when the page type is MyContent".
+This won't appear on pages whose type is any other model, because it might not make sense there.
+
+Now, stick this just before the `{% endblock %}` in your `content/mycontent.html` template:
+
+```
+{% for section in pages.current.contentsection_set.all() %}
+<section>
+  <h2>{{ section.title }}</h2>
+
+  {% if section.text %}
+    {{ section.text|html}}
+  {% endif %}
+</section>
+{% endfor %}
+```
+
+And that concludes part 1: You can now build pages out of an arbitrary number of sections
+In fact, for a lot of sites, you might not even need to write a single view!
+
+## Let's make another content model: a deeper dive
+
+For the second part of this walkthrough, we are going to create a simple blog app using some of the CMS's more advanced tooling.
+
+First, create an app called `news`, add it to your `INSTALLED_APPS`, and add this to your `models.py`:
```

Comment:
Just clarify `news/models.py`. If you say just `your models.py` it could be interpreted and the `models.py` from the previous step